src/main/kotlin/org/opensearch/commons/alerting/action/IndexCommentRequest.kt

```diff
@@ -0,0 +1,57 @@
+package org.opensearch.commons.alerting.action
+
+import org.opensearch.action.ActionRequest
+import org.opensearch.action.ActionRequestValidationException
+import org.opensearch.core.common.io.stream.StreamInput
+import org.opensearch.core.common.io.stream.StreamOutput
+import org.opensearch.rest.RestRequest
+import java.io.IOException
+
+class IndexCommentRequest : ActionRequest {
+    val entityId: String
+    val commentId: String
+    val seqNo: Long
+    val primaryTerm: Long
+    val method: RestRequest.Method
+    var content: String
+
+    constructor(
+        entityId: String,
+        commentId: String,
+        seqNo: Long,
```

Comment:
Could you add javadoc comments to explain the difference between `entityId` and `commentId`? Also, I'm not too clear on what `seqNo`, and `primaryTerm` are used for, so explaining that would be helpful too.